newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -1,4 +1,10 @@
 // @flow
+/*
+  When to launch a new search :
+    -> enter pressed (searchResultsDirty : false)
+    -> search button pressed
+    -> searchResultsDirty is true, and then make it false.
```

Comment:
Let's rather have a comment in the code itself. It has too much chance to be forgotten when there is an update if it's a the top of the file.

Reply:
Will do that!